src/workflows/workflows_test.ts

```diff
@@ -74,6 +99,8 @@ Deno.test("WorkflowStep export input values", () => {
   assertEquals(`${step1Inputs.email}`, "{{inputs.email}}");
   assertEquals(`${step1Inputs.name}`, "A name: {{inputs.name}}");
   assertEquals(`${step1.outputs.url}`, "{{steps.0.url}}");
+  assertEquals(`${step1.outputs.manager?.email}`, "{{steps.0.manager.email}}");
```

Comment:
I noticed this when testing that the output parameter reference is treated as optional here, and shouldn't be (these will always be set for parameter references, it's runtime where it's optional). I'll open up an issue for what needs fixed here, I think it's a slight tweak to the `WorkflowParameterReferences` type.

Reply:
Issue created - #48